libraries/ms-common/inc/extra_unity.h

```diff
@@ -1,5 +1,11 @@
 #pragma once
+
+#include <stdint.h>
+
 #include "status.h"
 #include "unity.h"
 
 #define TEST_ASSERT_OK(code) TEST_ASSERT_EQUAL(STATUS_CODE_OK, (code))
+
+// TEST ONLY FUNCTION TO SET TIMER COUNTER FOR SOFT TIMERS UNSAFE TO CALL OUTSIDE A TEST.
+void TEST_SOFT_TIMER_SET_COUNTER(uint32_t counter_value);
```

Comment:
Unless there get to be a lot of them (which I doubt) I don't think it is worth the file bloat to have individual files for each module. For naming, I think TEST should prefix the function to make it clear it is only to be used in tests. As for the rest of the name, I could care less. I renamed the file.